### DISCUSSION TITLE
Pull changes from Xpad repo to Pop! kernel 21.04

### DISCUSSION THREAD

**Discussion by ruineka** [pop-os/pop#1970] [Q&A]
I submitted a fix getting the gamepad on the OneXPlayer working on the paroj/xpad github that was merged this morning. Can these fixes be pulled into Pop! so people using the OneXPlayer will have a working gamepad out of the box?

**Comment by jacobgkau** [comment]
The title of your discussion includes the word "kernel." Is the change you're wanting part of the kernel, or the xpad package?

If it's the kernel, you can open a pull request on https://github.com/pop-os/linux.

If it's the xpad package, that package is currently coming from Ubuntu and is not being repackaged by Pop!_OS (you can see this by running `apt policy xpad`), and it would make the most sense to try and get that backported in upstream Ubuntu.
